### DISCUSSION TITLE
user-guide/install

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#123] [文档评论区]
# user-guide/install

主程序（PC） 请在 Zotero 官网 (https://zotero.org) (https://zotero.org) 下载对应自己操作系统的安装包。 image.png 浏览器插件 Zotero Connector Zotero 相较于其他文献管理软件的一大优势就是可以直接从网页中抓取元数据并存入数据库，这个过程通过浏览器插件 Zotero C...

https://zotero-chinese.com/user-guide/install.html

**Comment by Anphetamine** [comment]
zotero已经出了官方的Android app客户端，目前已经上架googleplay，目前还是beta版本

**Reply by northword** [reply]
感谢提醒。

![image](https://github.com/zotero-chinese/wiki/assets/44738481/afcec33c-3592-49cf-a869-57aa39426d73)

在另一页面有提到，我们会也在这个页面更新。

测试版本功能尚不完善，所以页面暂时没有提供测试版本的链接，正式发布后若国内下载不变将会提供。

**Comment by xujiakun-bit** [comment]
使用zotero 7正式版，很吃运行内存，8G运存经常到93%以上，请问有什么办法解决吗

**Reply by northword** [reply]
- 少装插件
- 少同时打开 pdf 等
- 用 64 位的 Zotero
- 加装内存

**Comment by Bella00412** [comment]
你好，在edge下载zotero connector显示download interrupt是为什么？如何解决？

**Comment by mouhorse** [comment]
请问zotero6和7怎么同时登陆不同账号，我在6里面登录某个账号，发现7里面也自动登录了这个账号。

**Comment by heidimyjoy** [comment]
mac系统上safari拓展插件不能抓取文献是什么原因？

**Reply by heidimyjoy** [reply]
主要是知网文献抓取失败，但是换edge浏览器上就可以。

**Comment by zhouzino** [comment]
dev-debug 102版本有水印，这个正常吗？怎么去掉水印？

**Comment by E2580** [comment]
PC端怎么设置开启自启动

**Comment by 520max** [comment]
我的zoero双击打不开，已经尝试过重启电脑和卸载重装了，请问还有什么方法打开吗？

**Comment by xinfa1** [comment]
华为的硅基流动邀请码：qDLMR0ps  邀请链接：https://cloud.siliconflow.cn/i/qDLMR0ps
注：通过分享邀请链接或者在注册时候输入邀请码，每邀请一人，双方均可以获取2000w字符量的API调用，永久有效。欢迎互帮互助呀

**Comment by AJUN1601** [comment]
友友们，为什么注册页面没有显示输入框呢？
